resources/user-guides/attach-data.md

```diff
@@ -181,24 +186,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target request objects to Target to prefetch content events
+### Attaching additional data Target to prefetch content events<a name="Target-Attach-Data-Prefetch-Content"></a>
 
 The following sample shows how to add a custom mbox to prefetch in all outgoing `prefetchContent` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule-prefetch)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event-prefetch)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action-prefetch)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property-prefetch)
+1. [Create a new **Rule**](#target-create-rule-prefetch)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event-prefetch)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action-prefetch)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-prefetch)
```

Comment:
Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-prefetch)."